Jenkinsfile

```diff
@@ -114,13 +115,13 @@ dockerBuild {
     } // stage
 
     stage('Terraforming') {
-        if (env.BRANCH_NAME == 'geocat_3.4.x') {
+        if (env.BRANCH_NAME.endsWith("auto-deploy") || env.BRANCH_NAME == "geocat_3.4.x") {
           executeInContainer(deployContainerName, """cd /terraform-geocat &&
             ln -s /root/bin/terraform /usr/bin             &&
             make install                                   &&
             make init                                      &&
             cd rancher-environments/geocat-dev             &&
-            terraform apply""")
+            terraform apply -var geocat_tag=${dockerTag}""")
```

Comment:
This will require (but it might likely already be the case) that the terraform repository has already parameterized the `geocat_tag`

Reply:
See https://github.com/camptocamp/terraform-geocat/commit/6e2cc85f5218e9c5ddcfd52be3b51819624c8ec8